### PR TITLE
[hi] Localize api-group.md

### PR DESCRIPTION
**Pull request by OtherHorizon**
Signed-off-by: Shaurya Ayush [](mailto:shauryachoudhary2015@gmail.com)

Localizes `content/en/docs/reference/glossary/api-group.md` to Hindi `content/hi/docs/reference/glossary/api-group.md`

Mentioned Issue:- #35600

### CODE REVIEW

**Comment by sftim** [comment]
/hold

pending the fix to not change `.gitignore` (please squash out any interim commit that does touch that file)

File: content/hi/docs/reference/glossary/api-group.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: API समूह
+id: API-समूह
+date: 2019-09-02
+full_link: /docs/concepts/overview/kubernetes-api/#api-groups-and-versioning
+संक्षिप्त_विवरण: >
```
Comment (sftim) [inline content/hi/docs/reference/glossary/api-group.md:6]:
```suggestion
short_description: >
```

**Comment by divya-mohan0209** [comment]
Hello @OtherHorizon , thank you for your efforts. However, we already have a localized version of the API Group file merged under [this PR](https://github.com/kubernetes/website/pull/35962). I will close this one off since we do not want to duplicate efforts.

/close

**Pull request closed without merging**